hypothesis-python/src/_hypothesis_pytestplugin.py

```diff
@@ -14,33 +14,39 @@
 # END HEADER
 
 import base64
+import sys
 from inspect import signature
 
 import pytest
 
-from hypothesis import HealthCheck, Phase, Verbosity, core, settings
-from hypothesis.errors import InvalidArgument
-from hypothesis.internal.detection import is_hypothesis_test
-from hypothesis.internal.escalation import current_pytest_item
-from hypothesis.internal.healthcheck import fail_health_check
-from hypothesis.reporting import default as default_reporter, with_reporter
-from hypothesis.statistics import collector, describe_statistics
-
```

Comment:
It would be good to add a comment explicitly explaining why the pytest plugin module has such a weird name, to save future readers from having to dig through history on their own.